cli/app.go

```diff
@@ -2748,6 +2748,60 @@ This won't work unless you have an existing installation of our GitHub app on yo
 				},
 			},
 		},
+		{
+			Name:            "inference",
+			Usage:           "work with cloud hosted inference service",
+			UsageText:       createUsageText("inference", nil, false, true),
+			HideHelpCommand: true,
+			Subcommands: []*cli.Command{
+				{
+					Name:  "infer",
```

Comment:
Since there's also one command, what do you think about it just being `viam infer`?

Reply:
I actually don't think so :) I think the other functionalities for the near future will be purely internal (e.g. setting up an endpoint or calling inferences on multiple images), so I would prefer to just have the single command.